### PR TITLE
fix: add browser build for webpack 5

### PR DESCRIPTION
**Pull request by AlexVarchuk**
## What/Why/How?
fixed https://github.com/Redocly/redoc/issues/1584
## Check yourself

- [x] Code is linted
- [x] Tested
- [ ] All new/updated code is covered with tests

### CODE REVIEW

**Review by Oprysk — APPROVED** [review]

File: webpack.config.ts [hunk 1]
```diff
@@ -47,13 +47,13 @@ export default (env: { standalone?: boolean } = {}) => ({
     fallback: {
       path: require.resolve('path-browserify'),
       http: false,
-      fs: false,
-      os: false,
+      fs: env.browser ? path.resolve(__dirname, 'src/empty.js') : false,
```
Comment (RomanHotsiy) [inline webpack.config.ts:50]: I believe we don't need this conditional here, just keep it empty. It will be overwritten by `externalsPresets` anyways.

**Review by RomanHotsiy — APPROVED** [review]
LGTM 👍

**Comment by CarltonHowell** [comment]
Looks like it's still busted in Webpack 5, getting the error in release candidate 59:
```
config.js:65 Uncaught (in promise) TypeError: path.join is not a function
```

**Comment by RomanHotsiy** [comment]
Can you share a minimal reproducible sample?

**Pull request merged**